### ISSUE TITLE
Review how triggers behave when all of their triggers have already been triggered

### ISSUE THREAD

**Issue by Sataniel98** [DRE2N/DungeonsXL#782]
E.g. trigger sign 1 is triggered before game starts -> Signs with T1 that aren't initialized should be triggered on initialization

**Comment by ehetteNandaYo** [comment]
Is this related ?  creating a sign Trigger  1 R, says is not written correclty
![image](https://user-images.githubusercontent.com/62739770/81444251-d70d5400-9144-11ea-83ec-8abf2cd46799.png)

**Comment by Sataniel98** [comment]
@Nextopi No, the ID is probably already used somewhere. You can leave the second line empty and it will generate one.
